scripts/list-tasks/main.go

```diff
@@ -0,0 +1,157 @@
+// The list-tasks script enumerates all tasks using any variant of a particular
+// connector, and add them to the active flowctl draft.
```

Comment:
In my experiences of editing a whole bunch of task configs in a very manual way, I have found that it generally doesn't work very well to do a lot of them in the same publication, because one of them will inevitably fail validation for some reason (often because of some unrelated issue with a task that the validation expands to) and that will cause the entire publication to fail. Doing them in smaller chunks is usually what I've had to resort to.

I could see a functionality like "give me N configs where this feature flag isn't already set" being useful for doing smaller batches at a time. I see there's a name prefix included already that would help to accomplish the small batches part as it is.

Reply:
This is a very good point.

I can see the appeal of adding a "where the feature flag is unset" filter option to the task listing script, but using that to try and break things down into small batches feels like it would still make the whole process a lot more hands-on than I'd prefer.

I'm currently considering implementing a third helper script which can take the directory full of specs on disk and iteratively add them to drafts and publish the drafts, either individually or in small batches.